Description: Always create new config file in configpath
 Using locate() can lead to an attempt to write a new config file
 to a directory without write permission. The correct path is
 configpath (~/.config/higan).
Author: Tobias Hansen <thansen@debian.org>

--- a/higan/target-tomoko/configuration/configuration.cpp
+++ b/higan/target-tomoko/configuration/configuration.cpp
@@ -63,5 +63,6 @@
 }
 
 auto Settings::save() -> void {
-  file::write(locate("settings.bml"), BML::serialize(*this));
+  directory::create({Path::config(), "higan/"});
+  file::write({Path::config(), "higan/", "settings.bml"}, BML::serialize(*this));
 }
